keyboards/naked60/keymaps/default/rules.mk

```diff
@@ -0,0 +1,2 @@
+MOUSEKEY_ENABLE = yes
```

Comment:
Please delete this file or erase its entire contents.

Setting rules in the default keymap that are not present at the keyboard/revision level means compiled firmware will have different features enabled depending on whether it was compiled from a cloned repository or from QMK Configurator.